jsonschema-core.xml

```diff
@@ -551,15 +607,22 @@ Link: <http://example.com/my-hyper-schema#>; rel="describedby"
 
             <section title='Describing a profile of JSON' anchor="profile">
                 <t>
-                    Instances MAY specify a "profile" as described in <xref target="RFC6906">The 'profile' Link Relation</xref>.
-                    When used as a media-type parameter, HTTP servers gain the ability to perform Content-Type Negotiation based on profile.
-                    The media-type parameter MUST be a whitespace-separated list of URIs (i.e. relative references are invalid).
+                    Instances MAY specify a "profile" as described in
+                    <xref target="RFC6906">The 'profile' Link Relation</xref>.
+                    When used as a media-type parameter, HTTP servers gain the ability to perform
+                    Content-Type Negotiation based on profile.
+                    The media-type parameter MUST be a whitespace-separated list of URIs (i.e.
```

Comment:
It seems that if a line ends in a period then the .txt generator automatically puts two spaces after that period.  It can't tell the difference between an abbreviation and the end of a sentence.

Could you please move the "(i.e." down to the beginning of the next line?